README.rst

```diff
@@ -98,3 +98,40 @@ End-user
 You just wanted to have a look at the application and found a bug? Please tell
 us and go fill a new issue:
 https://github.com/spiral-project/ihatemoney/issues
+
+How to release?
+===============
+
+In order to prepare a new release, we are following the following steps.
+
+The `prerelease` and `postrelease` commands are coming from `zest.releaser
+<https://pypi.python.org/pypi/zest.releaser>`_.
+
+Install `zest.releaser`:
+
+.. code-block:: bash
+
+    $ pip install "zest.releaser"
+
+- Merge remaining pull requests
+- Update ``CHANGELOG.rst``
+- Update :file:`CONTRIBUTORS.rst`. The following hairy command will output the full list:
+
+.. code-block:: bash
+
+     $ git shortlog -sne | awk '{$1=""; sub(" ", ""); print}' | awk -F'<' '!x[$1]++' | awk -F'<' '!x[$2]++' | sort
```

Comment:
This one liner does not find my actual name *« Jocelyn Delalande <jocelyn@crapouillou.net> »*